velox/dwio/parquet/tests/ParquetTableScanTest.cpp

```diff
@@ -47,42 +123,50 @@ class ParquetTableScanTest : public HiveConnectorTestBase {
     split->fileFormat = dwio::common::FileFormat::PARQUET;
     return split;
   }
+
+  std::shared_ptr<const RowType> rowTypes;
+  std::unordered_map<std::string, uint32_t> colNameMappings;
+  std::vector<std::shared_ptr<connector::ConnectorSplit>> splits;
 };
 
 TEST_F(ParquetTableScanTest, basic) {
-  auto data = makeRowVector(
+  // Plain select
```

Comment:
should we add tests for `filer + aggregate` also?

Reply:
Added the test